modules/installation/pages/proxy-setup.adoc

```diff
@@ -194,58 +195,59 @@ For more information about Cobbler, see xref:client-configuration:cobbler.adoc[C
+====
+
+Here is a detailed procedure step-by-step:
 
 [[proxy.migration3.replace]]
-.Procedure: Replacing a {susemgrproxy} and Keeping the Clients Registered
+.Procedure: Replacing a Proxy and Keeping the Clients Registered
 . Before starting the actual migration procedure, save the data from the old proxy, if needed.
-Consider copying important data to a central place that can also be accessed by the new proxy.
-// REMARK: 2019-08-23, ke: this is unclear
-// ** Copy the scripts that are still needed.
-// ** Copy the activation keys from the previous server.
-// Of course, it is always better to re-create the keys.
+Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
 . Shut down the proxy.
-. Install a new {susemgrproxy}{nbsp}{productnumber}, following xref:install-proxy-unified.adoc[Proxy Installation].
-. In the {productname} {webui} select the newly installed {susemgrproxy} and delete it from the systems list.
-[[step.at.proxy.migration3.replace.react]]
+. Install a new {productname} Proxy{nbsp}{productnumber}, following xref:install-proxy-unified.adoc[Proxy Installation].
+. In the {productname} {webui} select the newly installed {productname} Proxy and delete it from the systems list.
 . In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
-Then click [guimenu]``Generate New Key``, and remember it (write it on a piece of paper or copy it to the clipboard).
+Click [guimenu]``Generate New Key``, and make a note of the new key, as you will need it later.
 For more information about reactivation keys, see xref:reference:systems/system-details/sd-reactivation.adoc[Reactivation Keys].
-. After the installation of the new proxy, perform the following actions (if needed):
-** Copy the centrally saved data to the new proxy system.
-** Install any other needed software.
-** If the proxy is also used for autoinstallation, do not forget to setup TFTP synchronization.
+. After the installation of the new proxy, you might also need to:
+** Copy the centrally saved data to the new proxy system
+** Install any other needed software
+** Set up TFTP synchronization if the proxy is used for autoinstallation
 
 [IMPORTANT]
-.Proxy Installation and Client Connections
 ====
 During the installation of the proxy, clients will not be able to reach the {productname} Server.
-After a {susemgrproxy} system has been deleted from the systems list, all clients connected to this proxy will be (incorrectly) listed as `directly connected` to the {productname} Server.
-After the first successful operation on a client _such as execution of a remote command or installation of a package or patch_ this information will automatically be corrected.
+After a proxy system has been deleted from the systems list, all clients connected to this proxy will be (incorrectly) listed as `directly connected` to the {productname} Server.
```

Comment:
```suggestion
After a proxy system has been deleted, the systems list shows all clients that were connected to the deleted proxy, as now being directly connected to the {productname} Server.
This is incorrect.
```

Reply:
Yes, thanks, this is better. But I fear it is still confusing.  I think this is only temporarily and it is only incorrectly displayed or communicated to the user. The configuration itself is correct, and this will be visible to the user after a certain period of time.

Do we need a note or different wording, @Loquacity ?

Reply:
Yeah, maybe. What about this instead? It moves the important information first, adds "temporarily" to make it clear it's not permanent, and splits the sentence up.

```suggestion
After you have a deleted a proxy, the systems list can be temporarily incorrect.
All clients that were previously connected to the proxy will show as being directly connected to the server instead.
```